examples/olsconfig.yaml

```diff
@@ -32,6 +32,13 @@ ols_config:
     lib_log_level: warning
   default_provider: my_bam
   default_model: ibm/granite-13b-chat-v2
+# question_filters:
+# - name: ip_filter
+#   regular_expression: '(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
```

Comment:
strictly speaking it is not IP (can match 999.999.999.999), but yes, it's not the point of this PR

Reply:
strictly speaking, this doesn't cover IPv6 addresses either, but that's not the point ;)